infrastructure/deployment/deploy.sh

```diff
@@ -269,6 +269,34 @@ split_and_join() {
    echo $SPLIT
 }
 
+pull_images_from_file() {
+    local file="$1"  # The file containing the Docker images
+    # Check if the file exists
+    if [[ ! -f "$file" ]]; then
+        echo "File not found: $file"
+        exit 1
+    fi
+    # Read each line (image name) from the file
+    while IFS= read -r image; do
+    {
+        image=$(echo "$image" | xargs)
+        # Skip empty lines and comments (lines starting with #)
+        if [[ -z "$image" || "$image" =~ ^# ]]; then
+            continue
+        fi
+        echo "Pulling $image..."
+        if docker pull "$image"; then
```

Comment:
On which machine does this image get pulled to? I looked at the images on the farajaland-dev machine right after deployment

```
riku@farajaland-dev:~$ docker images | grep fara
opencrvs/ocrvs-farajaland                       <none>                              5d511c748fe0   About an hour ago   1.54GB
opencrvs/ocrvs-farajaland                       3534c7a                             4e85e3bed115   About an hour ago   1.54GB
```

A moment later (docker swarm downloads images automatically later on in the process if images are not present). First one is the one that got deployed.

```
riku@farajaland-dev:~$ docker images | grep fara
opencrvs/ocrvs-farajaland                       <none>                              af7798ce0120   23 minutes ago      1.54GB
opencrvs/ocrvs-farajaland                       <none>                              5d511c748fe0   About an hour ago   1.54GB
opencrvs/ocrvs-farajaland                       3534c7a                             4e85e3bed115   About an hour ago   1.54GB
```